mlflow/store/abstract_store.py

```diff
@@ -214,12 +219,14 @@ def list_run_infos(self, experiment_id, run_view_type):
     @abstractmethod
     def log_batch(self, run_id, metrics, params, tags):
         """
-        Logs multiple metrics, params, and tags for the specified run
+        Log multiple metrics, params, and tags for the specified run
+
         :param run_id: String id for the run
         :param metrics: List of :py:class:`mlflow.entities.Metric` instances to log
         :param params: List of :py:class:`mlflow.entities.Param` instances to log
         :param tags: List of :py:class:`mlflow.entities.RunTag` instances to log
-        :returns Tuple (failed_metrics, failed_params, failed_tags) where each element of
+
+        :return: Tuple (failed_metrics, failed_params, failed_tags) where each element of
```

Comment:
The return value is actually wrong here (I think you'd pointed this out to me), it should just be `:return: None`. If you want to change it in this PR that'd be great, otherwise happy to do it in a follow-up